src/components/molecules/NavBar/NavBar.scss

```diff
@@ -164,15 +164,25 @@ $border-radius: 28px;
   white-space: nowrap;
   text-overflow: ellipsis;
   transition: transform 400ms $transition-function;
+  border: none;
+  appearance: none;
+  -webkit-appearance: none;
+  color: white;
   img {
     height: 40px;
   }
-  &:hover {
+  &:hover,
+  &:focus {
+    outline: none;
+    border: none;
```

Comment:
Why settings these to `none`?

Don't remove `outline`, it causes a11y issues not fixes them.